### DISCUSSION TITLE
FoundationDB as Storage Backend: still feasible and advisable ?

### DISCUSSION THREAD

**Discussion by raphael10-collab** [JanusGraph/janusgraph#3640] [Q&A]
I'm interested to know the state of the JanusGraph-FoundationDB Storage Adapter:

4 years ago it was announced : https://www.youtube.com/watch?v=rQM_ZPZy8Ck&list=PLU8TPe7k8z9ew5W6YpACnGvjBDYaJVORZ&index=1

and, according to the `README` : https://github.com/JanusGraph/janusgraph-foundationdb the storage adapter is compatible only for the `JanusGraph` version `0.5.2`  with `FoundationDB` version `6.2.22`, while the latest `JanusGraph` version is `0.6.3` and the latest `FoundationDB` version is `7.2` (https://apple.github.io/foundationdb/ )

Since I don' see `FoundationDB` listed in storage backends 
![image](https://user-images.githubusercontent.com/70313067/222767462-ec428624-4a8d-4fa6-bb9a-f9e7635397f9.png), and there is no official JanusGraph documentation about FoundationDB as Backend Storage, 
I would like to know if it is still feasible, and, more importantly, advisable, to use `FoundationDB` as `storage backend` for `JanusGraph`

**Comment by FlorianHockmann** [comment]
You've posted the same question in three different places: JanusGraph/janusgraph-foundationdb#62, #1048 [(comment)](https://github.com/JanusGraph/janusgraph/issues/1048#issuecomment-1453771597) and here. Please don't do that. It won't make it more likely that you get a response, but it will annoy people following the project as they now get three notifications for the same issue instead of one.
I posted a response to your questions in JanusGraph/janusgraph-foundationdb#62.
